src/pages/_app.tsx

```diff
@@ -1,52 +1,57 @@
 import { type AppType } from "next/dist/shared/lib/utils";
 import { AragonProvider } from "@daobox/use-aragon";
-
+import { ToastContainer } from "react-toastify";
 import { WagmiConfig, goerli } from "wagmi";
+import "react-toastify/dist/ReactToastify.min.css";
 
 import "@styles/globals.css";
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
 import "@rainbow-me/rainbowkit/styles.css";
 import { client as wagmiClient, chains } from "../lib/wagmiClient";
-import { RainbowKitProvider, darkTheme, lightTheme } from "@rainbow-me/rainbowkit";
+import {
+  RainbowKitProvider,
+  darkTheme,
+  lightTheme,
+} from "@rainbow-me/rainbowkit";
 import { AppShell } from "../components/layout/AppShell";
 import { useEffect, useState } from "react";
 import { useReadLocalStorage } from "usehooks-ts";
 
 const MyApp: AppType = ({ Component, pageProps }) => {
   const [mounted, setMounted] = useState(false);
-  const mode = useReadLocalStorage("theme")
+  const mode = useReadLocalStorage("theme");
   function currentMode() {
```

Comment:
im sure all this logic can be abstracted away into a custom hook. there is too much going on in this _app file